base/socket.jl

```diff
@@ -243,13 +243,9 @@ InetAddr(ip::IPAddr, port) = InetAddr{typeof(ip)}(ip, port)
 type TCPSocket <: LibuvStream
     handle::Ptr{Void}
     status::Int
-    line_buffered::Bool
```

Comment:
do those packages realize this parameter has been ignored by the IO system since shortly before the v0.1 release?